en/django_start_project/README.md

```diff
@@ -118,6 +118,22 @@ ALLOWED_HOSTS = ['127.0.0.1', '.pythonanywhere.com']
 
 > Also add `.amazonaws.com` to the `ALLOWED_HOSTS` if you are using cloud9
 
+> If you are hosting your project on `Glitch.com`, let us protect the Django secret key that needs to 
+> remain confidential (otherwise, anyone remixing your project could see it) :
+>   * Create a file ```.env``` at the root of your project and add the following property in it :
+>     {% filename %}.env{% endfilename %}
+>     ```bash
+>     # Remember to change the value below to any, very long, very random sequence of characters
+>     SECRET='a-completely-random-string-of-letters-numbers-and-special-characters-anything-goes'
+>     ```
+>   * Then update the Django settings file to inject this secret value and set the Django web site name :
+>     {% filename %}mysite/settings.py{% endfilename %}
+>     ```python
+>     SECRET_KEY = os.getenv('SECRET')
+>     # and a little further, we inject the name of your new Glitch website ...
+>     ALLOWED_HOSTS = [os.getenv('PROJECT_DOMAIN') + ".glitch.me"]
+>     ```
```

Comment:
If these aren't consecutive lines in `settings.py`, format it as two separate code blocks.

Reply:
Thanks, I created two distinct list items.